### PR TITLE
adds files configuration to rspec plugin

### PR DESCRIPTION
**Pull request by mkristian**

### CODE REVIEW

File: jruby-gradle-rspec-plugin/src/main/groovy/com/github/jrubygradle/rspec/RSpec.groovy [hunk 1]
```diff
@@ -23,12 +23,19 @@ class RSpec extends DefaultTask {
     String version = DEFAULT_VERSION
 
     @Input
+    String files
```
Comment (rtyler) [inline jruby-gradle-rspec-plugin/src/main/groovy/com/github/jrubygradle/rspec/RSpec.groovy]: I thought this was going to be called "pattern"?

**Comment by rtyler** [comment]
This failed in Travis because these plugins are using the crappy torquebox proxy:

> ```
> 17:40:50.947 [Test worker] INFO  o.g.i.r.t.http.HttpClientHelper - Failed to get resource: GET. [HTTP HTTP/1.1 502 Proxy Error: http://rubygems-proxy.torquebox.org/releases/rubygems/rspec/3.3.0/rspec-3.3.0.gem]
> ```

Will you update the repo to hit rubygems.lasagna.io/proxy/releases instead?

**Comment by mkristian** [comment]
the reason for the torquebox repo is the base plugin version. does it make more sense to wait until the base plugin eats lasagna.io and then update the version of the base plugin dependency ?

**Comment by rtyler** [comment]
@mkristian aha, that's fine, I'll go ahead and merge away

**Pull request merged**